lib/ASTGen/Sources/ASTGen/Macros.swift

```diff
@@ -189,3 +190,20 @@ public func evaluateMacro(
     return 0
   }
 }
+
+/// Calls the given `allMacros: [Any.Type]` function pointer and returns a newly
+/// allocated buffer containing metadata pointers.
+@_silgen_name("swift_ASTGen_getMacroTypes") // SWIFT_CC
```

Comment:
It's not critical, but we've been using `@_cdecl` everywhere else in ASTGen so that we don't need the SWIFT_CC hacks.